#! /bin/sh /usr/share/dpatch/dpatch-run
## 06_ftbfs_fix.dpatch by Michael Casadevall <sonicmctails@gmail.com>
##
## DP: Corrects FTBFS by adding missing header

@DPATCH@
diff -urNad alsaplayer-0.99.80~/alsaplayer/CorePlayer.h alsaplayer-0.99.80/alsaplayer/CorePlayer.h
--- alsaplayer-0.99.80~/alsaplayer/CorePlayer.h	2007-07-08 10:15:55.000000000 -0400
+++ alsaplayer-0.99.80/alsaplayer/CorePlayer.h	2008-08-30 07:29:54.000000000 -0400
@@ -31,6 +31,10 @@
 #include "AlsaSubscriber.h"
 #include "input_plugin.h"
 
+#ifdef __linux__
+#include <linux/limits.h>
+#endif // __linux__
+
 // Tunable parameters
 
 #define BUF_SIZE (10240)	// Size of a single ringbuffer partition
